docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 PyYAML==6.0.2
-ruff==0.7.4
+ruff==0.8.0
 mkdocs==1.6.1
-mkdocs-material==9.1.18
+mkdocs-material==9.5.46
```

Comment:
I think we'll want to keep this in sync with the insider version as well. Currently, our insider version is at a commit where the public version matches `9.5.38`, so can you update it accordingly:

* `mkdocs-material` (public): `9.5.38`
* `mkdocs-material` (insider): 	`39da7a5e761410349e9a1b8abf593b0cdd5453ff`